Dockerfile

```diff
@@ -0,0 +1,27 @@
+FROM nvidia/cuda:12.0.0-base-ubuntu22.04
```

Comment:
Is an Nvidia image necessary for the base zkWASM repo? Does zkWASM have GPU support?